packages/thumbprint-react/package.json

```diff
@@ -39,6 +39,7 @@
         "@thumbtack/thumbprint-tokens": "^8.0.2",
         "classnames": "^2.2.6",
         "date-fns": "^1.28.5",
+        "focus-trap": "^5.0.1",
         "focus-trap-react": "^6.0.0",
```

Comment:
Seems like even the latest version of focus-trap-react (6.0.0) still uses an older version of focus-trap (4.x). I can downgrade focus-trap to 4.x in our usage, but I feel like it would be better to use the latest version of the library in our supported components, and move towards deleting ModalStandard, ModalBase, and focus-trap-react.

Reply:
Downgraded to 4.x